src/chi/components/accordion/accordion.scss

```diff
@@ -66,6 +66,16 @@ $icon-sizes: (
     }
   }
 
+  &__wrapper {
```

Comment:
By wrapper I was suggesting renaming .chi-accordion to .chi-accordion__item and containing all accordion items in .chi-accordion. This would provide more control over style and allow us to target the last child in the series to remove the bottom border

Reply:
```
<div class="chi-accordion__item">
  <div class="chi-accordion">
    <button class="chi-accordion__trigger">
      <i class="chi-icon icon-chevron-down"></i>
      <div class="chi-accordion__title">Accordion 1</div>
    </button>
    <div class="chi-accordion__content">
      <p class="chi-accordion__text">Content goes here</p>
    </div>
  </div>

  <div class="chi-accordion">
    <button class="chi-accordion__trigger">
      <i class="chi-icon icon-chevron-down"></i>
      <div class="chi-accordion__title">Accordion 1</div>
    </button>
    <div class="chi-accordion__content">
      <p class="chi-accordion__text">Content goes here</p>
    </div>
  </div>
</div>
```
Is this what you mean?